### PR TITLE
SQLIdentifier support isKeyword

### PR DESCRIPTION
**Pull request by vkill**
The sql `INSERT INTO "table" ("column", "updatedAt") VALUES ($1, $2) ON CONFLICT ("column") DO UPDATE SET "updatedAt" = EXCLUDED."updatedAt"`

Plan A: Use SQLLiteral.numeric, but numeric is inaccurate.

```
.numeric("""
EXCLUDED."updatedAt"
""")
```

Plan B: SQLIdentifier support isKeyword. in this case, EXCLUDED is a virtual table
```
.column(
    .table(.identifier("EXCLUDED", isKeyword: true)),
    .identifier("updatedAt")
)
```

### CODE REVIEW

File: Sources/SQL/SQLLiteral.swift [hunk 1]
```diff
@@ -2,11 +2,12 @@ public protocol SQLLiteral: SQLSerializable {
     associatedtype DefaultLiteral: SQLDefaultLiteral
     associatedtype BoolLiteral: SQLBoolLiteral
     static func string(_ string: String) -> Self
+    static func rawString(_ string: String) -> Self
```
Comment (tanner0101) [inline Sources/SQL/SQLLiteral.swift]: I like this change, but I wonder if we should call this something like `keyword` or `sql`? That could potentially be a more obvious difference than `rawString` vs. `string`.

Comment (vkill) [reply Sources/SQL/SQLLiteral.swift]: Hi, I think SQLIdentifier support isKeyword is better. what do you think?

**Comment by tanner0101** [comment]
Is `EXCLUDED` a PostgreSQL-specific thing or is it a SQL standard? If it is just for Postgres, I think we should add it as a special case to `PostgreSQLTableIdentifier` since it identifies a table. That will allow it to be used anywhere a table name can be used in Postgres. And also won't add any unnecessary complexity to the other SQLs if they don't support it.

**Comment by vkill** [comment]
Yes, you are right, I will submit a new PR in postgresql project.

**Pull request closed without merging**